exporter/awsemfexporter/util.go

```diff
@@ -21,54 +21,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
```

Comment:
This import should happen after the rest.

Reply:
I ran goimports, Ill give it another go but all lint checks passed